ChaosMod/game.h

```diff
@@ -4,22 +4,22 @@
 
 enum class GameVersion
 {
-    GTA5_LEGACY,
-    GTA5_ENHANCED
+	GTA5_LEGACY,
+	GTA5_ENHANCED
 };
 
 inline GameVersion GetGame()
 {
-    static GameVersion result = getGameVersion() > 1000 ? GameVersion::GTA5_ENHANCED : GameVersion::GTA5_LEGACY;
-    return result;
+	static GameVersion result = getGameVersion() > 1000 ? GameVersion::GTA5_ENHANCED : GameVersion::GTA5_LEGACY;
```

Comment:
Wouldn't a better check for this be the executable name?

Legacy: GTA5.exe
Enhanced: GTA5_Enhanced.exe

Could also check the PDB path? Enhanced contains "gen9" in it while legacy does not.